src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+  background-color: #000;
+}
+
+.section {
+  padding: 20px;
+}
+
+.box {
+  border: 1px solid #e95d49;
+}
+
+.blue_box {
+  background-color: #308dc0;
+  margin: 20;
+  width: 350px;
+  box-sizing: border-box;
```

Comment:
let's add box-sizing: border-box rule to all the elements on the page (use * selector)